.travis.yml

```diff
@@ -0,0 +1,32 @@
+env:
+  global:
+    - CC_TEST_REPORTER_ID=
```

Comment:
Code for this is:  `0784251df8a033e7ebd699583cea39dbe393080594ad9217506d81417cd9d2df`

Reply:
```suggestion
    - CC_TEST_REPORTER_ID=0784251df8a033e7ebd699583cea39dbe393080594ad9217506d81417cd9d2df
```